File: app/src/main/java/com/jackpan/stockcomputer/model/StockEpsListData.java
package com.jackpan.stockcomputer.model;

import com.google.gson.annotations.SerializedName;

public class StockEpsListData {
    @SerializedName("排名")
    public String rank =  null ; ;

    @SerializedName("代號")
    public String codename= null ; ;

    @SerializedName("名稱")
    public String name= null ; ;

    @SerializedName("成交")
    public String deal= null ; ;

    @SerializedName("財報年度")
    public String financialyear= null ; ;

    @SerializedName("營收(億)")
    public String revenue= null ; ;

    @SerializedName("營收成長(%)")
    public String revenuegrowth= null ; ;

    @SerializedName("毛利(億)")
    public String grossprofitbillion= null ; ;

    @SerializedName("毛利成長(%)")
    public String grossprofitgrowth= null ; ;

    @SerializedName("淨利(億)")
    public String netincomebillion= null ; ;

    @SerializedName("淨利成長(%)")
    public String netincomegrowth= null ; ;

    @SerializedName("毛利(%)")
    public String grossprofit= null ; ;

    @SerializedName("毛率增減")
    public String grossprofitiod= null ; ;

    @SerializedName("淨利(%)")
    public String netincome= null ; ;

    @SerializedName("淨率增減")
    public String netincomeiod= null ; ;

    @SerializedName("EPS(元)")
    public String eps= null ; ;

    @SerializedName("EPS增減(元)")
    public String epsiod= null ; ;

    @SerializedName("ROE(%)")
    public String roe= null ; ;

    @SerializedName("ROE增減")
    public String roeiod= null ; ;

    @SerializedName("ROA(%)")
    public String poa= null ; ;

    @SerializedName("ROA增減")
    public String poaiod= null ; ;

    @SerializedName("財報評分")
    public String score= null ; ;
}

File: app/src/main/java/com/jackpan/stockcomputer/model/StockEpsListData.kt
package com.jackpan.stockcomputer.model

import com.google.gson.annotations.SerializedName

data class StockEpsListData(
    @SerializedName("排名")
    var rank : String,
    @SerializedName("代號")
    var codename : String,
    @SerializedName("名稱")
    var name : String,
    @SerializedName("成交")
    var deal : String,
    @SerializedName("財報年度")
    var financialyear : String,
    @SerializedName("營收(億)")
    var revenue : String,
    @SerializedName("營收成長(%)")
    var revenuegrowth : String,
    @SerializedName("毛利(億)")
    var grossprofitbillion : String,
    @SerializedName("毛利成長(%)")
    var grossprofitgrowth : String,
    @SerializedName("淨利(億)")
    var netincomebillion : String,
    @SerializedName("淨利成長(%)")
    var netincomegrowth : String,
    @SerializedName("毛利(%)")
    var grossprofit : String,
    @SerializedName("毛率增減")
    var grossprofitiod : String,
    @SerializedName("淨利(%)")
    var netincome : String,
    @SerializedName("淨率增減")
    var netincomeiod : String,
    @SerializedName("EPS(元)")
    var eps : String,
    @SerializedName("EPS增減(元)")
    var epsiod : String,
    @SerializedName("ROE(%)")
    var roe : String,
    @SerializedName("ROE增減")
    var roeiod : String,
    @SerializedName("ROA(%)")
    var poa : String,
    @SerializedName("ROA增減")
    var poaiod : String,
    @SerializedName("財報評分")
    var score : String
    )